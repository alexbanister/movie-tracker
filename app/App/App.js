import React, { Component } from 'react';
import CardCatelog from '../CardCatelog/CardCatelog';

import { Header } from '../Header/Header';
import { Route } from 'react-router-dom';
import { Login } from '../Login/login';
import { SignUp } from '../SignUp/SignUp';


export default class App extends Component {

  render() {
    return ([
      <Route path='/' component={Header} key='header'/>,
      <Route exact path='/' component={CardCatelog} key='CardCatelog'/>,
      <Route exact path='/login' component={Login} key='login' />,
      <Route exact path='/signup' component={SignUp} key='signup' />

    ]);
  }
}
