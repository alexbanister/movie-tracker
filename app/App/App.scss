* {
  box-sizing: border-box;
}
body {
  margin: 0;
  padding: 0;
  font-family: 'Lato', sans-serif;
  color: $off-white;
  background: url('http://localhost:3000/assets/background.png');
  background-position: center;
  width: 100vw;
  height: 100vh;
}
a {
  text-decoration: none;
}
#root {
  @include linear-gradient(rgba(0,0,0,.9), rgba(0,0,0,.5));
  min-height: 100vh;
}
