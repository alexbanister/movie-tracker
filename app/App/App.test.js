import { shallow } from 'enzyme';
import App from './App';
import React from 'react';

describe('App snapshot', () => {

  it('should always match the snapshot', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
