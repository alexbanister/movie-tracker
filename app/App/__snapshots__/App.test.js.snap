// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot should always match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Array [
    <Route
      component={[Function]}
      path="/"
    />,
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />,
    <Route
      component={[Function]}
      exact={true}
      path="/favorites"
    />,
    <Route
      component={[Function]}
      exact={true}
      path="/login"
    />,
    <Route
      component={[Function]}
      exact={true}
      path="/signup"
    />,
  ],
  Symbol(enzyme.__nodes__): Array [
    Array [
      <Route
        component={[Function]}
        path="/"
      />,
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />,
      <Route
        component={[Function]}
        exact={true}
        path="/favorites"
      />,
      <Route
        component={[Function]}
        exact={true}
        path="/login"
      />,
      <Route
        component={[Function]}
        exact={true}
        path="/signup"
      />,
    ],
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
